sprint/.ddev/docker-compose.sprint.yml

```diff
@@ -0,0 +1,43 @@
+version: "3"
+
+services:
+  ide:
+    container_name: ddev-${DDEV_SITENAME}-ide # This is the name of the container. It is recommended to follow the same name convention used in the main docker-compose.yml file.
+    image: briangilbert/cloud9-alpine:20180318
+    ports:
+    - 8000 # Coud9 is served from this port inside the container
+    environment:
```

Comment:
I was a bit confused at first because previously had been setting USERNAME and PASSWORD env vars to "drupal". Documentation follow-up to make sure those are aware it's the default username:password (pending Brian's issue of not having to specify it at all).

Reply:
the user/pass gets echoed out later, also had previously tried setting the env vars via the docker compose file and it didn't appear to be working